base/special/bessel.jl

```diff
@@ -147,6 +147,33 @@ for jy in ("j","y"), nu in (0,1)
     end
 end
 
+"""
+    besselj0(x)
+
+Bessel function of the first kind of order 0, ``J_0(x)``.
+"""
+besselj0(x)
```

Comment:
If there's only a single docstring for `besselj0` then either syntax is fine really. It'll only matter if we had several different methods of the function and we wanted to document each one separately.